errs/errors.go

```diff
@@ -19,8 +19,9 @@ import (
 )
 
 var (
-	_ ErrorWrapper = &Error{}
-	_ StackError   = &Error{}
+	_ ErrorWrapper  = &Error{}
+	_ StackError    = &Error{}
+	_ fmt.Formatter = &Error{}
```

Comment:
Figured it'd be good to explicitly document/verify that the interface is implemented on Error as well